src/components/CustomPagination.tsx

```
@@ -24,29 +25,33 @@ const CustomPagination: React.FC<CustomPaginationProps> = ({
+            <InfiniteScroll
+                dataLength={items.length}
+                next={fetchMoreData || (() => { 
+                    console.warn('fetchMoreData callback is required for infinite scroll');
+                })}
+                hasMore={hasMore}
+                loader={items?.length >= TotalCount ? '' : <h4>{t('COMMON.LOADING')}...</h4>}
+                // endMessage={
+                //     <p style={{ textAlign: 'center' }}>You have seen all data!</p>
+                // }
+            >
+                <></>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)